libs/eo/landing-page/shell/src/lib/eo-landing-page-header.component.ts

```diff
@@ -74,15 +80,33 @@ const selector = 'eo-landing-page-header';
       class="${selector}__toolbar watt-space-inset-squished-m"
     >
       <img src="assets/energyorigin-logo.svg" alt="EnergyOrigin" />
-      <a mat-button mat-flat-button routerLink="/login">Start</a>
+      <a
+        mat-button
+        mat-flat-button
+        *rxLet="loginUrl$ as loginUrl"
+        [href]="loginUrl"
+        >Start</a
+      >
```

Comment:
praise: Great! This is a decent way to render the link for what we need right now. Notice that the link is not rendered until we have a response which makes sense to our requirements right now.